Cleanup of Section 1

lachellel opened the pull request:
clean up of section 1, removed delegation, put restrictions into enterprise RA, adopted formats from other CP documents, and added placeholders for identifying the CAs and policy objects

lachellel committed:
clean up of section 1, removed delegation, put restrictions into enterprise RA, adopted formats from other CPs and placeholders for identifying the CAs and policy objects

lachellel committed:
self assessments 

fixes #119

lachellel committed:
cps approval procedures

fixes #24

konklone commented on certificate-policy.md:
```diff
@@ -65,82 +70,82 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 * Ensuring continued conformance of each CA that issues certificates under this policy with applicable requirements as a condition for allowing continued participation, and
 * Ensuring compliance with CAB Forum Baseline Requirements
 
+The Federal PKI Policy Authority SHALL review and update this Certificate Policy at least every 365 days to ensure compliance with CAB Forum Baseline requirements.  After review and approval, the document version number and a dated changelog entry shall be added even if no changes were deemed necessary. 
```
Since CA/B ballots have an expected time-to-implement of much less than 365 days, I think we might want to say: 

> The Federal PKI Policy Authority SHALL review and update this Certificate Policy at least every 365 days, and within X days of any new CAB Forum Baseline requirements.

lachellel replied:
I'm hesitant to put "within X days" because not all CAB Forum Baseline requirement updates may impact or apply.  For example: delegated third party updates wouldn't apply to **every** CA infrastructure for even non-profit or commercial CAs.

For traditional FPKI - we have a 30 day timeframe to incorporate FPKI change proposals that are accepted and voted upon into the policies, with implementation timeframes applied.  

For CA/B requirements, we've looked at every update for the past 2++ years and tracked the requirement changes as 'applicable' or 'not applicable'.   Can we consider a statement that is a bit broader than within X days of **any** new CAB Forum requirement?  Since the policy applies to the root and issuing CAs in a **hierarchical** PKI, the policy and **practices / CPS** need to be updated in tandem.

lachellel replied:
I moved this to Section 2.0 - there were duplicates stated over and over ...added language for when to update and flowdowns? 

https://github.com/uspki/policies/pull/147/commits/60c2d521b1bcbae27540118e060d870e50019c50

konklone replied:
:+1:

konklone commented on certificate-policy.md:
```diff
@@ -65,82 +70,82 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 
 ### 1.5.2 Contact Person
-Contact information for the Federal Public Key Infrastructure Policy Authority is **TODO: Insert Contact Info**.
+Contact information for the Federal Public Key Infrastructure Policy Authority is fpki@gsa.gov.
```
Small thing, but I think it'd be worth making a new alias for this one (which could include fpki@gsa.gov as a subscriber).

aakunz replied:
Suggest adding a statement in Section 1.0 Overview to the effect that this policy incorporates Certificate Transparency as a key component in providing a publicly accessible and accountable PKI.
This sets the stage up front and provides an anchor for additional references to CT later in the policy.

lachellel replied:
@aakunz added the sentence to the very first paragraph / intro.

https://github.com/uspki/policies/pull/147/commits/6bbb2d9f43aa8b2dff0913cd6213aa0cf23887e9

lachellel replied:
agree but can we open an issue on the last pass through, and merge?  I think we need more contact info and statement of government (etc) besides just an email address too.

konklone commented on certificate-policy.md:
```diff
@@ -65,82 +70,82 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 ### 1.4.1 Appropriate Certificate Uses
-The primary goal of these Requirements is to enable efficient and secure electronic communication, while addressing user concerns about the trustworthiness of Certificates. These Requirements also serve to inform users and help them to make informed decisions when relying on Certificates. This Certificate Policy and requirements for US Federal Government (USG) are limited to publicly trusted Devi ...
-
+This Certificate Policy (CP) and requirements for U.S. Federal Government is limited to Publicly Trusted Device Certificates used for identifying and authenticating services accessible on the Internet. 
```
So for "accessible on the Internet" - why would we want to forbid agencies from using this on intranets, as long as they're sufficiently aware that hostnames and  certs become public?

It also might incur us some audit risk if our certs get used on services that were internet-accessible at validation time, but have since become publicly inaccessible.

lachellel replied:
I'm looking for a statement that clearly delineates that this is not for only locally resolveable.  And that it's not for IoT or any of the myriad applications of PKI in closed network environments or only locally trusted.

konklone commented on certificate-policy.md:
```diff
@@ -65,82 +70,82 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
-RA's verified Domain Namespace.
-2. If the certificate request includes a Subject name of a type other than a Fully-Qualified Domain Name, the CA SHALL confirm that the name is either that of the delegated enterprise, or an Affiliate of the delegated enterprise, or that the delegated enterprise is an agent of the named Subject. For example, the CA SHALL NOT issue a Certificate containing the Subject name of "Agency1" on the auth ...
+1. The CA SHALL confirm that the requested Fully-Qualified Domain Name(s) are within the Enterprise RA's verified Domain Namespace(s) as registered in the .GOV (DotGov) and .MIL (DotMil) gTLDs Domain Name Registrars.  
+2. The CA SHALL confirm that the requested Fully-Qualified Domain Name(s) are not within any Domain Namespace(s) for any U.S. State, Local, Territorial, Native Sovereign Nations, or any other entities identified as a _Non-Federal Agency_ in the DotGov Domain Name Registrar per United States Code (U.S.C.) 41 CFR Part 102-173. 
```
I again want to note that I see it as all-disadvantage for us to include the state/local/territorial/tribal limitation in the CP/CPS, even if we expect it as an operational limitation for the foreseeable future. 

It doesn't pose any risk to the web PKI for us to issue properly domain-validated certs to SLTT, and from the perspective of the community, they have to accept our root with the understanding that there's nothing technically constraining us from issuing for a SLTT .gov anyway.

lachellel replied:
I'll update this part to match the modified info from other sections.  The constraint is on our boundaries and not overstepping those boundaries from a public law, funding, and perception aspect. 

The best middle ground is making it a should versus a shall...intent is established.

lachellel replied:
updated in https://github.com/uspki/policies/pull/147/commits/ffa38b8bfd957371b7a172aadcdd1baee655ff38

along with other RA items

konklone commented on certificate-policy.md:
```diff
@@ -411,25 +416,29 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 # 2. PUBLICATION AND REPOSITORY RESPONSIBILITIES
-The Federal PKI Policy Authority SHALL annually update this CP to ensure compliance with CAB Forum Baseline requirements.
+The Federal PKI Policy Authority SHALL review and update this CP at least every 365 days to ensure compliance with CAB Forum Baseline requirements.
```
Same comment as above.

lachellel replied:
updated and moved everything into 2.0 for publication of CP, CPS, timeframes, etc. 

https://github.com/uspki/policies/pull/147/commits/60c2d521b1bcbae27540118e060d870e50019c50

konklone commented on certificate-policy.md:
```diff
@@ -411,25 +416,29 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
+The Federal PKI Policy Authority SHALL develop, implement, enforce, and annually update a Certificate Policy that describes how the CA implements the latest version of these Requirements. The review and update shall include any changes needed to address: 
+
+- US Federal Government mission needs and changes to procedures to support the missions
+- CAB Forum Baseline Requirements and updates 
```
Same comment as above. We will need to be prepared to update our documents more frequently than 365 days, if CA/B ballots pass that affect their operation.

lachellel replied:
moved into 2.0
https://github.com/uspki/policies/pull/147/commits/60c2d521b1bcbae27540118e060d870e50019c50

konklone reviewed (changes requested):
I left some comments that would be worth discussing, on the call at least.

lachellel committed:
simple ct add to intro

added ct to intro

lachellel committed:
updates to enterprise ra

clarification of boundaries
persons
SLTT

lachellel committed:
cp updates

cp updates and proposed timelines - moved all to Section 2.0, rather than have duplication
updated section 2.x in this branch for publishing artifacts (including CA certificates, CRLs, CPs, CPSs)
updated hosting of webpages

lachellel commented:
let's merge :rocket:

@konklone

konklone commented:
@lachellel There's a merge conflict. Mind addressing that? :+1: to you self-merging after that if the changes weren't somehow substantive.

lachellel commented:
Yep, i knew there would be one.  the trust store PR had overlaps...just a sec..

lachellel committed:
Merge branch 'master' into Section1-updates
